src/main/java/org/symphonyoss/integration/webhook/salesforce/parser/v2/SalesforceMetadataParser.java

```diff
@@ -68,123 +57,6 @@ public Message parse(Map<String, String> parameters, JsonNode node) throws Sales
-
-    if (!StringUtils.isEmpty(ownerEmail) && emailExistsAtSymphony(ownerEmail)) {
-      ((ObjectNode) node).put(SalesforceConstants.HAS_OWNER_AT_SYMPHONY, Boolean.TRUE);
-    }
-  }
-
-  private boolean emailExistsAtSymphony(String emailAddress) {
-    if (StringUtils.isBlank(emailAddress)) {
-      return false;
-    }
-
-    User user = userService.getUserByEmail(integrationUser, emailAddress);
-    return user.getId() != null;
-  }
-
-  protected void processAmount(JsonNode node) {
-    String amount = node.path(SalesforceConstants.AMOUNT).asText(EMPTY);
-
-    if (!StringUtils.isEmpty(amount)) {
-      amount = NumberFormatUtils.formatValueWithLocale(Locale.US, amount);
-
-      ((ObjectNode) node).put(SalesforceConstants.AMOUNT, amount);
-    } else {
-      ((ObjectNode) node).put(SalesforceConstants.AMOUNT, DEFAULT_VALUE_NULL);
-    }
-  }
-
-  protected void processCurrencyIsoCode(JsonNode node) {
-    String currencyIsoCode = node.path(SalesforceConstants.CURRENCY_ISO_CODE).asText(EMPTY);
-
-    formatOptionalField(node, SalesforceConstants.CURRENCY_ISO_CODE, currencyIsoCode);
-  }
-
-  protected void processCloseDate(JsonNode node) {
-    String closeDateFormat = node.path(SalesforceConstants.CLOSE_DATE).asText(null);
-    SimpleDateFormat formatter = new SimpleDateFormat(SalesforceConstants.TIMESTAMP_FORMAT);
-
-
-    if (!StringUtils.isEmpty(closeDateFormat)) {
-      try {
-        closeDateFormat = formatter.format(formatter.parse(closeDateFormat));
-      } catch (ParseException e) {
-        // Do nothing
-      }
-
-      ((ObjectNode) node).put(SalesforceConstants.CLOSE_DATE, closeDateFormat);
-    } else {
-      ((ObjectNode) node).put(SalesforceConstants.CLOSE_DATE, DEFAULT_VALUE_NULL);
-    }
-  }
-
-  protected void processNextStep(JsonNode node) {
-    String nextStep = node.path(SalesforceConstants.NEXT_STEP).asText(EMPTY);
-
-    formatOptionalField(node, SalesforceConstants.NEXT_STEP, nextStep);
-  }
-
-  protected void processStageName(JsonNode node) {
-    String stageName = node.path(SalesforceConstants.STAGE_NAME).asText(EMPTY);
-
-    formatOptionalField(node, SalesforceConstants.STAGE_NAME, stageName);
-  }
-
-  protected void processProbability(JsonNode node) {
-    String probability = node.path(SalesforceConstants.PROBABILITY).asText(EMPTY);
-
-    formatOptionalField(node, SalesforceConstants.PROBABILITY, probability);
-  }
-
-  protected void proccessURLIconIntegration(JsonNode node) {
-    String urlIconIntegration = getURLFromIcon("salesforce.svg");
-
-    if (!urlIconIntegration.isEmpty()) {
-      ((ObjectNode) node).put(SalesforceConstants.URL_ICON_INTEGRATION, urlIconIntegration);
-    }
-  }
-
-  protected void proccessIconCrown(JsonNode node) {
-    String iconCrown = getURLFromIcon("new_opportunity.svg");
-
-    if (!iconCrown.isEmpty()) {
-      ((ObjectNode) node).put(SalesforceConstants.ICON_CROWN, iconCrown);
-    }
-  }
-
   protected String getURLFromIcon(String iconName) {
```

Comment:
This method could hold the whole logic regarded to Icon URL and it's JSON handling.